test/unit/any_instance_method_test.rb

```diff
@@ -66,7 +55,7 @@ def test_should_include_the_filename_and_line_number_in_exceptions
     method.define_new_method
 
     expected_filename = 'stubbed_method.rb'
-    expected_line_number = 60
+    expected_line_number = 46
```

Comment:
Wonder if we can make it less dependent on the exact line number? This broke a few times since I made multiple changes to StubbedMethod.

Reply:
Yeah, it is a bit annoying. I think I've tried one or two things in the past, but they haven't worked out.